### PR TITLE
Prod 326: deckinfo hi fi

### PR DESCRIPTION
**Pull request by yash431garg**
- Description
Updated deck info design
- What are the steps to test that this code is working?
New deck info card design
- Screen shots or recordings for UI changes
Screenshot 2024-09-19 at 7 28 33 PM
Screenshot 2024-09-19 at 7 30 40 PM

### CODE REVIEW

**Comment by yash431garg** [comment]
Please review this as well.

cc: @Yihwan @gatorthana

**Review by gatorthana — COMMENTED** [review]
hey @yash431garg ! Everything looks fixed except for the icon being square. If this takes more than 15min to fix, we can skip it for now! 
image

**Comment by yash431garg** [comment]
> hey @yash431garg ! Everything looks fixed except for the icon being square. If this takes more than 15min to fix, we can skip it for now! image

Hi @gatorthana, I missed this. It won't take much time. Will push this.

File: app/components/PreviewDeckCard/PreviewDeckCard.tsx [hunk 1]
```diff
@@ -40,13 +40,12 @@ const PreviewDeckCard = ({
               alt=""
               fill
               objectFit="cover"
-              className="rounded-lg overflow-hidden"
+              className="rounded-[8px] overflow-hidden"
```
Comment (marvinmarnold) [inline app/components/PreviewDeckCard/PreviewDeckCard.tsx:43]:
```suggestion
              className="rounded-full overflow-hidden"
```

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

**Review by gatorthana — COMMENTED** [review]
It's good to go, thank you! 🙏🏼

**Pull request merged**